### PR TITLE
Document Cloud SIEM Entities

### PR DESCRIPTION
**Pull request by jhgilbert**
### What does this PR do? What is the motivation?
DOCS-7860

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:2]: take out Cloud SIEM to align with the other guides - just "Risk Based Entity Insights"

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+{{< callout url=""https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
+Risk Based Entity Insights is in private beta.
+{{< /callout >}}
+
+## Overview
+
+[Cloud SIEM's Risk Based Entity Insights][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:11]: again add "for AWS" after entity insights for precision

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+With Risk Based Entity Insights, you can:
+
+- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
+- Triage relevant items in bulk.
+- Take mitigation steps such as creating an automated workflow or creating a global case for an entity.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:18]: Take mitigation steps such as creating **a global suppression** or creating a case for an entity.

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Explore Entities
+
+### Query and filter entities
+
+On the [Entities Explorer page][4], you can view all entities that have at least one signal.
+
+{{< img src="security/entities/entities-explorer.png" alt="A list of entities and their risk scores in the Entities Explorer" style="width:100%;" >}}
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:26]: can we use a screenshot that filters for severity and entity type instead? 
image

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+### Quickly build context on an entity
+
+Click an entity in the [Explorer][4] to open the entity details panel.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:30]: Click an entity in the [Explorer][4] to open the entity side panel.

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+{{< img src="security/entities/entity-side-panel.png" alt="The details panel for an entity" style="width:90%;" >}}
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:32]: image
use something like this from demo org instead :)

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+The top of the panel displays the count of related signals, identity risks, and so on, with links to the list view of each.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:34]: The what happened displays the count of signals and how they have contributed to the risk score as well as any configuration risks potentially found.

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+The **Entity Context** section displays entity's attributes, such as the entity type and the list of fired signals.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:36]: The **What contributes** section displays the list of fired signals, relevant misconfigurations, and identity risks.

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+
+### Triage and mitigate threats in bulk
+
+The **Next steps** section of the entity details panel includes the available mitigation steps.
+
+{{< img src="security/entities/entity-next-steps.png" alt="The available next steps for an entity as shown in the entity details panel" style="width:80%;" >}}
+
+## Prerequisites
+
+- For coverage of AWS resources, [AWS must be configured for Cloud SIEM][1].
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:46]: For AWS entities coverage, [AWS must be configured for Cloud SIEM][1].

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+- To view associated Cloud Security Management (CSM) data in the entity panel, [CSM must be configured][2].
+
+No other setup is required to use Cloud SIEM Entities.
+
+## Risk scoring
+
+An entity's risk score summarizes the entity's risk level over time.
```
Comment (quachlingA) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:53]: An entity's risk score summarizes the entity's risk level over the past 14 days.

File: content/en/security/cloud_siem/entities_and_risk_scoring.md (hunk 6 above)
Comment (quachlingA) [reply content/en/security/cloud_siem/entities_and_risk_scoring.md:32]: hey jen sorry to bring this up again could you please use an entity side panel that has those high signals at the top (there is currently a bug that we are fixing but the impact on risk score should be sorted within the signals)

File: content/en/security/cloud_siem/entities_and_risk_scoring.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+
+The **What contributes** section displays the list of fired signals, relevant misconfigurations, and identity risks.
+
+### Triage and mitigate threats in bulk
+
+The **Next steps** section of the entity details panel includes the available mitigation steps.
+
+{{< img src="security/entities/entity-next-steps.png" alt="The available next steps for an entity as shown in the entity details panel" style="width:80%;" >}}
+
+## Prerequisites
```
Comment (estherk15) [inline content/en/security/cloud_siem/entities_and_risk_scoring.md:44]: Suggest moving this above **Explore entities**

**Review by estherk15 — APPROVED** [review]
Looks great!

**Pull request merged**